.eslintrc.json

```diff
@@ -20,7 +21,7 @@
     "no-case-declarations": "error",
     "no-catch-shadow": "off",
     "no-class-assign": "error",
-    "no-cond-assign": "error",
+    "no-cond-assign": "warn",
```

Comment:
What necessitated this change?

Reply:
This was required for the luhn check, as this is the [formula](https://github.com/CruPCI/cru-payments/blob/master/src/credit-card/card-number/utils/luhn-check.ts#L10):
 `counter += (odd = !odd) ? digit : luhnArray[digit];`

The newer version of esLint throws an error at this code.